keyboards/keebart/sofle_choc_pro/config.h

```diff
@@ -0,0 +1,19 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+#pragma once
+
+#define SERIAL_USART_TX_PIN GP12
+#define USB_VBUS_PIN GP13
+
+#define I2C_DRIVER I2CD1
+#define OLED_UPDATE_INTERVAL 50
+#define I2C1_SDA_PIN GP6
+#define I2C1_SCL_PIN GP7
+
```

Comment:
I guess based on the store listing provided in the `readme.md` file, this keyboard does not actually support OLED displays? All I2C configuration should be removed then.
```suggestion
```